src/main/java/ua/shpp/eqbot/command/CommandContainer.java

```diff
@@ -23,20 +24,28 @@ public class CommandContainer {
     @Autowired
     public CommandContainer(SendBotMessageService sendBotMessageService, UserRepository userRepository,
                             ServiceRepository serviceRepository, ProviderRepository providerRepository,
-                            ImageService imageService, BundleLanguage bundleLanguage, UserService userService) {
+                            ImageService imageService, BundleLanguage bundleLanguage, UserService userService,
+                            ProviderService providerService) {
         commandMap = ImmutableMap.<String, Command>builder()
                 .put(CommandName.REG.getNameCommand(), new RegistrationNewUser(sendBotMessageService, userService, bundleLanguage))
                 .put(CommandName.START.getNameCommand(), new StartCommand(sendBotMessageService, bundleLanguage))
                 .put(CommandName.HELP.getNameCommand(), new HelpCommand(sendBotMessageService))
                 .put(CommandName.NO.getNameCommand(), new NoCommand(sendBotMessageService))
                 .put(CommandName.SETTINGS.getNameCommand(), new SettingsCommand(sendBotMessageService))
-                .put(CommandName.CHANGE_ROLE_TO_PROVIDER.getNameCommand(), new ChangeRoleToProviderCommand(sendBotMessageService, providerRepository))
-                .put(CommandName.ADD_SERVICE.getNameCommand(), new AddService(sendBotMessageService, serviceRepository, imageService, providerRepository, userService, bundleLanguage))
-                .put(CommandName.SEARCH_SERVICE.getNameCommand(), new SearchService(sendBotMessageService, serviceRepository, providerRepository, userService, bundleLanguage))
-                .put(CommandName.DELETE_USER.getNameCommand(), new DeleteUserCommand(sendBotMessageService, userRepository, userService, bundleLanguage, providerRepository, serviceRepository))
-                .put(CommandName.MAIN_MENU.getNameCommand(), new MainMenu(sendBotMessageService))
+                .put(CommandName.CHANGE_ROLE_TO_PROVIDER.getNameCommand(),
+                        new ChangeRoleToProviderCommand(sendBotMessageService, providerRepository))
+                .put(CommandName.ADD_SERVICE.getNameCommand(),
+                        new AddService(sendBotMessageService, serviceRepository, imageService, bundleLanguage))
+                .put(CommandName.DELETE_USER.getNameCommand(), new DeleteUserCommand(sendBotMessageService,
```

Comment:
Lost a piece of code:
`.put(CommandName.SEARCH_SERVICE.getNameCommand(), new SearchService(sendBotMessageService, serviceRepository, providerRepository, userService, bundleLanguage))`